src/utils/base-route.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+
+/**
+ * stores BUSINESS_ID, TEMP_REG_NUMBER, BASE_URL, VUE_ROUTER_BASE from initial route.
+ */
+
+export function setBaseRouteAndBusinessId (pathname: string, processEnvBaseUrl: string, rootUrl: string): Promise<any> {
```

Comment:
I kinda like the file named the same as the function it exports. I don't mind long filenames :)

Reply:
Great, I like that too!